Cargo.toml

```diff
@@ -18,24 +18,28 @@ openapi = ["dep:utoipa"]
 
 [dependencies]
 tokio = { workspace = true, features = ["process"] }
+utoipa = { workspace = true, optional = true }
+stackable-operator = { workspace = true }
 k8s-openapi = { workspace = true }
 serde_json = { workspace = true }
 serde_yaml = { workspace = true }
 indexmap = { workspace = true }
 reqwest = { workspace = true }
 tracing = { workspace = true }
-utoipa = { workspace = true, optional = true }
+bcrypt = { workspace = true }
 snafu = { workspace = true }
 which = { workspace = true }
 serde = { workspace = true }
 tera = { workspace = true }
 kube = { workspace = true }
+rand = { workspace = true }
 url = { workspace = true }
 
 [build-dependencies]
 gobuild = "0.1.0-alpha.1"
 
 [workspace.dependencies]
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", rev = "b2ebbc7" }
```

Comment:
Why does this refer to a commit hash rather than a branch or tag?

Reply:
Changed in 939d5e3